src/compiler/scala/tools/nsc/transform/Mixin.scala

```diff
@@ -1001,8 +1001,13 @@ abstract class Mixin extends InfoTransform with ast.TreeDSL {
           val parents1 = currentOwner.info.parents map (t => TypeTree(t) setPos tree.pos)
           // mark fields which can be nulled afterward
           lazyValNullables = nullableFields(templ) withDefaultValue Set()
+          val bodyEmptyAccessors = if (!sym.enclClass.isTrait) body else body mapConserve {
+            case dd: DefDef if dd.symbol.isAccessor && !dd.symbol.isLazy =>
+              deriveDefDef(dd)(_ => EmptyTree)
+            case tree => tree
+          }
```

Comment:
I'm pretty sure that's fixed in my PR. Maybe we could get that one merged first, or rebase yours on top of it -- it's ready (also, I'm flying to Old Country on Wednesday night).